### PR TITLE
Includes more comparators: gt, ge, lt, le

### PR DESCRIPTION
**Pull request by dillonhuff**
@jeffsetter @rdaly525

### CODE REVIEW

File: src/simulator/interpret.cpp [hunk 1]
```diff
@@ -1109,23 +1111,52 @@ namespace CoreIR {
...
+            int rv = r.to_type<int>();
 
-          float lf = bitCastToFloat(lv);
-          float rf = bitCastToFloat(rv);
+            float lf = bitCastToFloat(lv);
+            float rf = bitCastToFloat(rv);
 
-          cout << "lf = " << lf << endl;
-          cout << "rf = " << rf << endl;
+            cout << "lf = " << lf << endl;
```
Comment (rdaly525) [inline src/simulator/interpret.cpp]: Looks like there are some debug statements here and elsewhere in this file. Could you remove them?

**Comment by dillonhuff** [comment]
@rdaly525 the printouts are now removed

**Comment by dillonhuff** [comment]
@rdaly525 also why is float.abs a binary operation? I assumed it was short for absolute value.

**Comment by rdaly525** [comment]
@dillonhuff, you are correct. That is a mistake and should be a unary

**Pull request merged**